src/pybert/__init__.py

```diff
@@ -10,8 +10,11 @@
 
 Copyright (c) 2014 by David Banas; All rights reserved World wide.
 """
-__version__ = "4.1.0"
-__date__ = "March 15, 2023"
+from importlib.metadata import version as _get_version
+
+# Set PEP396 version attribute
+__version__ = _get_version("PipBERT")  # PyPi "PyBERT" package name got stollen. :(
```

Comment:
When you install with conda, isn't the name still pybert?  Will this fail?  When I use setuptools_scm, it has a pattern like the following and maybe we can adapt that to check for both:

```python
from importlib.metadata import version, PackageNotFoundError

try:
    __version__ = version("PipBERT") # For PyPi installs
except PackageNotFoundError:
    try:
         __version__ = version("pybert")  # For conda installs.
    except PackageNotFoundError:
         __version__ = "Unknown Version"
```

Reply:
Cool trick!

Yes, the _Conda_ package is still named "pybert".
However, the `import lib.metadata.version()` function refers to the `...dist-info` file, which takes its name from the `pyproject.toml` file. And, there, the package is named "PipBERT".

I've tested _Conda_: build, install, and run, and everything works.